hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -323,12 +317,6 @@ def onchange_add_project_id(self):
                          self.timesheet_ids.mapped('task_id').ids)],
                 },
             }
-        else:
-            return {
```

Comment:
why?

Reply:
Because `add_line_task_id` is invisible when `add_line_project_id` is empty, so this part of code is not reachable.

Reply:
Invisible doesn't mean empty. It could be filled and still be invisible. Better assure the domain independently if it is invisible or not.

Reply:
:thinking: why do you need to set a different domain for a field that is invisible?

Reply:
I don't understand your point, sorry.

Reply:
Because maybe you have an extension where you don't have that field invisible.

Reply:
Ok, I'm going to restore that block of code then :+1:

Reply:
Well, yeah, don't worry. Minimize this observation.